bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -225,11 +234,12 @@ public CheckAndMutateRowResponse checkAndMutateRow(CheckAndMutateRowRequest requ
         // Only log for powers of two to avoid spam.
         int count = LOG_COUNT.incrementAndGet();
         if ((count & (count - 1)) == 0) {
-          LOG.info("Retries configured for non-retryiable request. Not retrying. " +
-              "In future releases this case will fail.");
+          LOG.info(
+              "Retries configured for non-retryiable request. Not retrying. "
+                  + "In future releases this case will fail.");
         }
       }
-      return channelPool.newCall(method, CallOptions.DEFAULT);
+      return rpc.call(request);
```

Comment:
Feels silly to construct the RetryableRpc and then not use it. Maybe check the retryiability before, probably in performAsyncRpc (specially if performBlockingRpc calls it).